Add pricing faq for PostHog AI credits (#13524)
PostHog/posthog.com: contents/docs/posthog-ai/pricing-faq.mdx

# PostHog AI credits

> 

Reviewer: ```suggestion
```

PostHog uses **AI credits** to charge for all of its AI features.  

> Each time you interact with an AI-powered feature — like asking PostHog to write SQL, summarize data, or analyze sessions — credits are used based on the effort required to complete your request.

Reviewer: We use line wrapping in the editor with `cmd + z` in vscode instead of manual next line positions

```suggestion
AI features in PostHog consume **AI credits**. Each time you interact with an AI-powered feature – like asking PostHog to write SQL, summarize data, or analyze sessions – credits are used based on the effort required to complete your request.
```

This is also probably a little annoying but we use `–` (en dashes) 😅

This means you only pay for what you actually use.  

> There’s no fixed subscription — and every organization has a free tier worth $25 (2,500 credits) so you can explore all PostHog’s AI features before paying anything.

Reviewer: ```suggestion
This means you pay for what you actually use. There’s no fixed subscription and every organization has a free tier worth $25 (2,500 credits) to explore PostHog AI's features.
```

### When AI credits are used

AI credits are consumed whenever PostHog performs intelligent work for you.  

> You’ll spot these features by the ✨ icon or when using the in-app chat.

Reviewer: ```suggestion
AI credits are consumed whenever PostHog performs intelligent work for you.  You’ll spot these features by the ✨ icon or when using the in-app chat.
```

@edwinyjlim maybe we rebase this on your [other PR](https://github.com/PostHog/posthog.com/pull/13614#pullrequestreview-3448256894) and link to the list of actions here (if we have a list)

Examples include:
- Searching for session recordings in natural language  
- Summarizing experiment results  
- Writing or debugging SQL with AI  

> ### What doesn't use credits

Reviewer: ```suggestion
## What doesn't consume credits
```

Getting help on how to use PostHog never costs credits.  
All AI features still in beta are also free to use — as of **Oct 31**, that includes **session summaries** and **deep research**.

> ### How credit usage works

Reviewer: ```suggestion
## How credit usage works
```

AI credits are based on the **underlying token costs**, which reflect the effort required to complete your request.

- Simple queries (like “What were my daily active users in October?”) use very few tokens, and therefore very few credits.  
- More complex tasks (like analyzing hundreds of session recordings or rewriting a SQL query multiple times) use more tokens and cost a bit more.  

While exact usage varies, credit consumption usually scales with complexity — more advanced tasks cost more but can deliver deeper insights and time savings.

> You’ll always see **real-time cost information** while using AI features.

Reviewer: ```suggestion
While exact usage varies, credit consumption usually scales with complexity – more advanced tasks cost more but can deliver deeper insights and time savings. You’ll always see **real-time cost information** while using AI features.
```

> PostHog automatically selects the most efficient model for each AI feature. We apply a simple, consistent 20% markup over the underlying LLM provider’s cost: So 1 PostHog AI credit equals $0.8333 of raw inference, and 1,000 credits cost $10.

Author: @gewenyu99 

Good spot, there are some 0's missing. It should be "For example, 1 PostHog AI credit equals $0.008333 of raw inference". Then it's $0.00833 + 20% markup = $0.01/credit.

| Example interaction | Estimated credit cost |
| --- | --- |
| “What are my daily active users in October?” | coming soon |
| “Show me session recordings for this organization_id” | coming soon |

> | “Can you help me write a SQL query that…” | coming soon |

Reviewer: ```suggestion
| "What are my daily active users in October?" | coming soon |
| "Show me session recordings for this organization_id" | coming soon |
| "Can you help me write a SQL query that…" | coming soon |
```

# PostHog AI credits

PostHog uses **AI credits** to charge for all of its AI features.  

> Each time you interact with an AI-powered feature — like asking PostHog to write SQL, summarize data, or analyze sessions — a small number of credits are used based on the complexity of your request.

Reviewer 2: Grammar:
```suggestion
Each time you interact with an AI-powered feature — like asking PostHog to write SQL, summarize data, or analyze sessions — a small number of credits is used based on the complexity of your request.
```

This means you only pay for what you actually use.  
There’s no fixed subscription — and every organization has a free tier worth $25 (2,500 credits) so you can explore all PostHog’s AI features before paying anything.

### **When AI credits are used**

AI credits are consumed whenever PostHog performs intelligent work for you.  
You’ll spot these features by the ✨ icon or when using the in-app chat.

Examples include:
- Searching for session recordings in natural language  
- Summarizing experiment results  
- Writing or debugging SQL with AI  

### **What doesn’t use credits**

> Getting help on how to use PostHog never costs credits.  

Reviewer 2: I don't think this is _strictly_ true, i.e. if you getting help involves a query being created – that costs credits.

Because we have a (we could say generous) free tier, is it perhaps then simpler – for both the user an us – to charge for the root node always? Regardless of whether just the docs search tool was involved, insight creation, or no tools at all. Especially with the new agent orchestration based on mode-switching, being prototyped by @skoob13, the tools become "dumb", and it's even trickier to say "this is just getting help" vs. "this is doing some work for the user".

All AI features still in beta are also free to use — as of **Oct 31**, that includes **session summaries** and **deep research**.

> ### **How credit usage works**

Reviewer 3: I agree this brings up questions like "well, why aren't they just charging me by tokens then?" 

That being said, I think this section is important to have. It just needs more detail and clarity and ideally a conversion calculator or formula. I'm assuming we're charging a slight markup or something!

> ### **How credit usage works**

Author: Clarified in the relevant section below

AI credits are based on **token costs**, which reflect the effort required to complete your request.

- Simple queries (like “What were my daily active users in October?”) use very few tokens, and therefore very few credits.  
- More complex tasks (like analyzing hundreds of session recordings or rewriting a SQL query multiple times) use more tokens and cost a bit more.  

While exact usage varies, credit consumption usually scales with value — more advanced tasks cost more but deliver deeper insights and time savings.

> You’ll always see **real-time cost information** while using AI features.

Reviewer 4: Will this real-time cost information include markup?

To keep it simple, token costs are converted into **AI credits**, billed at **$0.01 per credit** — so **1,000 credits = $10**.

| Example interaction | Estimated credit cost |
| --- | --- |
| “What are my daily active users in October?” | coming soon |
| “Show me session recordings for this organization_id” | coming soon |
| “Can you help me write a SQL query that…” | coming soon |

---

### **Track your AI credit usage**

Stay on top of your AI credit spend in real time:

1. Type `/usage` in chat to see the current cost of your conversation  

> 2. Check the **Billing & usage** page to view your total monthly consumption  

Reviewer 4: add links
- https://app.posthog.com/organization/billing/overview
- https://app.posthog.com/organization/billing/usage)

3. (Coming soon) A dashboard showing which AI features use the most credits  

> 💡 Credit data may take up to 24 hours to appear on the Billing & usage page.

---

### **Control your AI credit spend**

You’re always in control of your AI credit usage:
- **Real-time tracking** — see costs as you go  

> - **Billing limits** — set hard caps to prevent overspending  

Reviewer 4: Mention the default paid limit? Assuming we're going with it

> - **Usage alerts** — get notified when you hit key thresholds  

Reviewer 4: Are these the standard billing alerts when you hit 80% and 100% of your free allowance (if on free plan) or billing limit (if on paid plan)? Or something custom for AI?

Author: Yes, standard. We don't have anything custom planned for now
